machine/delta/delta_wb2448/kernel/acpi_enforce.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/drivers/acpi/osl.c b/drivers/acpi/osl.c
+index de03938..fe12708 100644
+--- a/drivers/acpi/osl.c
++++ b/drivers/acpi/osl.c
+@@ -1610,7 +1610,7 @@ __setup("acpi_no_auto_serialize", acpi_no_auto_serialize_setup);
+ #define ENFORCE_RESOURCES_LAX    1
+ #define ENFORCE_RESOURCES_NO     0
+ 
+-static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_STRICT;
++static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_NO;
```

Comment:
If we use ENFORCE_RESOURCES_STRICT, we will face below ACPI warning.
It will case i2c driver i801 init fail.

ACPI Warning: SystemIO range 0x000000000000E000-0x000000000000E01F conflicts with OpRegion 0x000000000000E000-0x000000000000E00F (\_SB_.PCI0.SBUS.SMBI) (20150410/utaddress-254)
ACPI: If an ACPI driver is available for this device, you should use it instead of the native driver